.github/workflows/beef-tool.yml

```diff
@@ -0,0 +1,29 @@
+name: Build beef-tool
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Install dependencies
+        run: |
+          pip install pyinstaller
+
+      - name: Package beef-tool
+        run: |
+          pyinstaller --onefile --noconsole beef-tool/beef-tool.py
+
+      - name: Upload executable
+        uses: actions/upload-artifact@v4
+        with:
+          name: beef-tool.exe
```

Comment:
The artifact is a zip file, so it gets named `beef-tool.exe.zip`. I would remove the `.exe` extension.